src/kbmod_wf/workflow.py

```diff
@@ -53,37 +53,18 @@ def create_uri_manifest(inputs=[], outputs=[], config={}, logging_file=None):
 @python_app(
     cache=True, executors=get_executors(["local_dev_testing", "small_cpu"]), ignore_for_cache=["logging_file"]
 )
-def read_and_log(inputs=[], outputs=[], logging_file=None):
-    """THIS IS A PLACEHOLDER FUNCTION THAT WILL BE REMOVED SOON"""
-    from kbmod_wf.utilities.logger_utilities import configure_logger
-
-    logger = configure_logger("task.read_and_log", logging_file.filepath)
-
-    with open(inputs[0].filepath, "r") as f:
-        for line in f:
-            value = line.strip()
-            logger.info(line.strip())
-
-    with open(outputs[0].filepath, "w") as f:
-        f.write(f"Logged: {value}")
-
-    return outputs[0]
-
-
-@python_app(
-    cache=True, executors=get_executors(["local_dev_testing", "small_cpu"]), ignore_for_cache=["logging_file"]
-)
-def uri_to_ic(inputs=[], outputs=[], logging_file=None):
+def uri_to_ic(inputs=[], outputs=[], runtime_config={}, logging_file=None):
     from kbmod_wf.utilities.logger_utilities import configure_logger
     from kbmod_wf.task_impls.uri_to_ic import uri_to_ic
 
     logger = configure_logger("task.uri_to_ic", logging_file.filepath)
 
     logger.info("Starting uri_to_ic")
     uri_to_ic(
```

Comment:
@wilsonbb Just to be clear, I've tried to organize the function parameters with the following order: 
- input filepaths
- output filepaths
- any other parameters (we should avoid these, and put them in the runtime config)
- runtime configuration dictionary
- logger

Does that seem like a reasonable ordering? 

I've also tried to be pretty explicit about the parameter names, opting for something descriptive like `uris_filepath` over `uris_file` just to be clear that we're passing a full path for instance instead of filename and a path separately. Thoughts on that?

Reply:
This approach seems reasonable to me!